File: Java/program/Human.java
// deklarasi kelas Human
public class Human {
    // atribut private
    private String nik, nama;
    private char jenis_kelamin;

    /* konstruktor */

    // konstruktor tanpa parameter
    public Human() {
        // set isi atribut secara default
        this.nik = "";
        this.nama = "";
        this.jenis_kelamin = '-';
    }

    // konstruktor dengan parameter
    public Human(String nik, String nama, char jenis_kelamin) {
        // set isi atribut secara default
        this.nik = nik;
        this.nama = nama;
        this.jenis_kelamin = jenis_kelamin;
    }

    /* Getter dan Setter */

    // Get nik
    public String getNik() {
        return this.nik;
    }

    // set nik
    public void setNik(String nik) {
        this.nik = nik;
    }

    // Get nama
    public String getNama() {
        return this.nama;
    }

    // set nama
    public void setNama(String nama) {
        this.nama = nama;
    }

    // get jenis_kelamin
    public char getJenis_kelamin() {
        return this.jenis_kelamin;
    }

    // set jenis_kelamin
    public void setJenis_kelamin(char jenis_kelamin) {
        this.jenis_kelamin = jenis_kelamin;
    }
}

File: CPP/program/Human.cpp
// import library
#include <iostream>
#include <string>

// menggunakan standard namespace
using namespace std;

// deklarasi kelas Human
class Human
{
    // atribut private
private:
    string nik, nama;
    char jenis_kelamin;

public:
    /* konstruktor */

    // konstruktor tanpa parameter
    Human()
    {
        // set isi atribut secara default
        this->nik = "";
        this->nama = "";
        this->jenis_kelamin = '-';
    }

    // konstruktor dengan parameter
    Human(string nik, string nama, char jenis_kelamin)
    {
        // set isi atribut berdasarkan isi parameter
        this->nik = nik;
        this->nama = nama;
        this->jenis_kelamin = jenis_kelamin;
    }

    /* Getter dan Setter*/

    // Get nik
    string get_nik()
    {
        return this->nik;
    }

    // Set nik
    void set_nik(string nik)
    {
        this->nik = nik;
    }
    // Get nama
    string get_nama()
    {
        return this->nama;
    }

    // Set nama
    void set_nama(string nama)
    {
        this->nama = nama;
    }

    // Get jenis_kelamin
    char get_jenis_kelamin()
    {
        return this->jenis_kelamin;
    }

    // Set jenis_kelamin
    void set_jenis_kelamin(char jenis_kelamin)
    {
        this->jenis_kelamin = jenis_kelamin;
    }

    /* desktruktor */

    // default
    ~Human() // desktruktor
    {
    }
};
